### PR TITLE
Fix name for CentOS and Ubuntu

### PR DESCRIPTION
**Pull request by piranna**
According to https://www.centos.org/, Centos must be written as CentOS. The same for https://ubuntu.com/, where "Ubuntu Linux" is found nowhere. This commit fix both of them.

### CODE REVIEW

**Comment by retrohacker** [comment]
Heya @piranna, great observation and thank you for the contribution :heart:. This PR LGTM. Happy to merge this as soon as the tests are passing. LMK if you need guidance on how to make that happen.

**Comment by piranna** [comment]
> Happy to merge this as soon as the tests are passing

Tests fixed :-) I've also fixed a duplicated entry for Mandrake distro.

File: os.json [hunk 1]
```diff
@@ -25,9 +25,8 @@
   "/etc/linuxppc-release" : ["Linux-PPC"],
   "/etc/mageia-release" : ["Mageia"],
   "/etc/mandriva-release" : ["Mandriva Linux", "Mandrake Linux"],
-  "/etc/mandrake-release" : ["Mandriva Linux", "Mandrake Linux"],
+  "/etc/mandrake-release" : ["Mandrake", "Mandriva Linux", "Mandrake Linux"],
```
Comment (retrohacker) [inline os.json]: It looks like we've duplicated `Mandrake` here and changed the order of detection would could break `Mandriva`. 

Could we merge `Mandrake Linux` and `Mandrake` and keep it at the end of the array?

**Comment by piranna** [comment]
> the `Mandrake` de-duplication was an awesome catch

There should be some checks about that, like jsonlint or something else.

> Unless we are shuffling the order because of a known bug (i.e. Mandrake is never detected) I'd prefer to keep it at the end of the array to keep things stable.

Good point, i'll do it.

File: os.json (hunk 1 above)
Comment (piranna) [reply os.json]: Done.

**Comment by retrohacker** [comment]
Thanks for the contribution! I've added you as a collaborator if you're interested.

**Comment by piranna** [comment]
> Thanks for the contribution! I've added you as a collaborator if you're interested.

Great, thank you :-)

**Comment by piranna** [comment]
When will this be published on npm?

**Comment by retrohacker** [comment]
Just published as 3.2.0

**Comment by piranna** [comment]
> Just published as 3.2.0

Thank you :-)

**Pull request merged**